templates/login.html

```diff
@@ -0,0 +1,27 @@
+{% extends "base.html" %}
+{% block title %}Home{% endblock %}
+{% block bodyclass %}page-login{% endblock %}
+{% block content %}
+  <canvas id="lemon-canvas" width="500" height="500"></canvas>
+
+  {% if show_login %}
+    <form method="post" action="/login" class="login-form">
+      <input
+        type="password"
+        placeholder="password"
+        name="password"
+        class="login-form-input"
+      />
+
+      <input type="submit" value="LOG IN" class="login-form-button" />
+
+      {% if login_error %}
+        <span class="login-form-error">wrong password!</span>
+      {% endif %}
+    </form>
+    <br />
```

Comment:
I think we should just use `<br>`: https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br